app/views/state_file/state_file_pages/about_page.html.erb

```diff
@@ -42,4 +42,13 @@
     </div>
   <% end %>
 </section>
-<% end %>
\ No newline at end of file
+<% end %>
+
+<div>
```

Comment:
not like it really matters but this div will be present but empty when the feature is not enabled. i'm not sure how long we intend to keep the feature flag but if it's going away when the feature launches (as in, we delete `if Flipper.enabled?...` etc) i wouldn't bother changing this